myuw/data/resource_link_import.csv

```diff
@@ -82,8 +82,8 @@ Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/olco/Secure/Ac
 Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/dining/,Campus Dining,http://www.uwb.edu/food,Campus Dining,http://www.tacoma.uw.edu/food-services/locations,Campus Dining,no
 Campus Life,Housing & Dining,all,https://www.hfs.washington.edu/myhfs/ledger.aspx,Check HFS Balance,,,,,,,no
 Campus Life,Housing & Dining,all,,,https://www.hfs.washington.edu/housing/,Student Housing,http://www.uwb.edu/housing,Student Housing,http://www.tacoma.uw.edu/housing,Housing,no
-Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/,Facilities Services,https://www.uwb.edu/facility,Facilities and Campus Operations,http://www.tacoma.uw.edu/facilities-services/facilities-services,Facilities Services,no
-Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/services,Facilities Services: Request Service,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,http://www.tacoma.uw.edu/facilities-services/work-order-request,Facilities Services: Work Order Request,no
+Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/,UW Facilities,https://www.uwb.edu/facility,Facilities and Campus Operations,http://www.tacoma.uw.edu/facilities-services/facilities-services,UW Facilities,no
+Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/services/requests,Submit Work Request,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,http://www.tacoma.uw.edu/facilities-services/work-order-request,UW Facilities: Work Order Request,no
```

Comment:
This appears to have changed the UWT link label, but it shouldn't have changed.
```suggestion
Campus Life,Housing & Dining,all,,,https://facilities.uw.edu/services/requests,Submit Work Request,http://dlweb.megamation.com/uwb/DLWEB.php/O4W_INFO_PAGE,Facilities Maintenance Request,http://www.tacoma.uw.edu/facilities-services/work-order-request,Facilities Services: Work Order Request,no
```

Reply:
Thanks for catching it! Changed both back.